phd-tests/framework/src/guest_os/mod.rs

```diff
@@ -11,30 +11,32 @@ use serde::{Deserialize, Serialize};
 
 mod alpine;
 mod debian11_nocloud;
+mod shell_commands;
 mod ubuntu22_04;
 mod windows;
+mod windows_server_2016;
 mod windows_server_2019;
 mod windows_server_2022;
 
 /// An entry in a sequence of interactions with the guest's command prompt.
 #[derive(Debug)]
-pub(super) enum CommandSequenceEntry {
+pub(super) enum CommandSequenceEntry<'a> {
     /// Wait for the supplied string to appear on the guest serial console.
-    WaitFor(&'static str),
+    WaitFor(Cow<'a, str>),
 
     /// Write the specified string as a command to the guest serial console.
-    WriteStr(&'static str),
+    WriteStr(Cow<'a, str>),
 
     /// Change the serial console buffering discipline to the supplied
     /// discipline.
     ChangeSerialConsoleBuffer(crate::serial::BufferKind),
 
     /// Set a delay between writing individual bytes to the guest serial console
     /// to avoid keyboard debouncing logic in guests.
```

Comment:
does this comment need to be updated, if the semantics of this have changed?

Reply:
Good catch, done. (In b5b8d81 I also cleaned up a few other comments that were invalidated by bb39823.)